### PR TITLE
Tweaks to Riot Android & iOS TMN and FAQ

### PR DESCRIPTION
**Pull request by aaronraimist**
Signed-off-by: Aaron Raimist <aaron@raim.ist>

### CODE REVIEW

File: content/faq/questions/q60.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### I installed Riot via F-Droid, why is it draining my battery?
 
-The [F-Droid](https://f-droid.org/repository/browse/?fdid=im.vector.alpha) release of Riot does not use [Google Cloud Messaging](https://developers.google.com/cloud-messaging/). This allows users that do not have or want Google Services installed to use Riot.
+The [F-Droid](https://f-droid.org/repository/browse/?fdid=im.vector.app) release of Riot does not use [Google Cloud Messaging](https://developers.google.com/cloud-messaging/). This allows users that do not have or want Google Services installed to use Riot.
```
Comment (babolivier) [inline content/faq/questions/q60.md]: Riot's F-Droid app ID is still `im.vector.alpha`

**Comment by aaronraimist** [comment]
If I resolve the merge conflicts can this get in? It's been sitting for a while and is still needed.

**Comment by benparsons** [comment]
> If I resolve the merge conflicts can this get in? It's been sitting for a while and is still needed.

@aaronraimist yes please do that

**Comment by aaronraimist** [comment]
Updated

**Pull request merged**